Deploy/deploycore.cpp

```diff
@@ -478,12 +478,12 @@ MSVCVersion DeployCore::getMSVC(const QString &_qtBin) {
         return static_cast<MSVCVersion>(res);
     }
 
-    if (!msvcPath.contains("msvc")) {
+    if (!msvcPath.contains("MSVC")) {
         QuasarAppUtils::Params::log("vcredis not defined");
         return static_cast<MSVCVersion>(res);
     }
 
-    auto base = msvcPath.mid(msvcPath.indexOf("msvc"), 11);
+    auto base = msvcPath.mid(msvcPath.indexOf("MSVC"), 11);
```

Comment:
Check please tests. The deploytest::testMSVC method is broken. You change the path of msvc but forget change this path on tests.

Reply:
> The deploytest::testMSVC method is broken. You change the path of msvc but forget change this path on tests.

Ok I fixed it in ea7439eebbb164017f39720b37f73c18850179a2 . But there are still a bunch of `added unnecessary file : C:/CQtDeployer_src/DistributionKit/vcredist_msvc2017_x64.exe`, so I hope you can help me with it.

Reply:
Yes i am describe this problem.